manifests/service.pp

```diff
@@ -11,6 +11,7 @@
   file { $service_file_location:
     content => template($service_file_template),
     mode    => '0755',
+    before  => Service[confluence],
```

Comment:
this will fail if someone doesn't manage the service.
we should move it into the service's require